feeds/calendar.ics

```diff
@@ -3,26 +3,23 @@ layout: none
 ---
 BEGIN:VCALENDAR
 VERSION:2.0
-PRODID:-//{{site.title}}//{{site.url}}//EN
+PRODID:-//{{ site.title }}//{{ site.url }}//EN
 CALSCALE:GREGORIAN
 METHOD:PUBLISH{% for post in site.posts %}{% if post.categories.last == "meetings-meetups" or post.categories.first == "events" or post.categories.first == "talks" or post.categories.first == "announcements" and post.layout == "post-event" %}
 BEGIN:VEVENT
-UID:{{site.url}}{{site.baseurl}}{{post.url}}
-DTSTAMP:{{ post.date | date: "%Y%m%dT000000Z" }}{% if post.date-start %}
-DTSTART;TZID=America/New_York:{{ post.date-start | date: "%Y%m%dT%H%M00" }}{% else %}
+UID:{{ site.url }}{{ site.baseurl }}{{ post.url | replace: '/', '-' }}
+DTSTAMP:{{ post.date | date: "%Y%m%dT%H%M%SZ" }}{% if post.date-start %}
+DTSTART;TZID=America/New_York:{{ post.date-start | date: "%Y%m%dT%H%M%S" }}{% else %}
 DTSTART:{{ post.date | date: "%Y%m%d" }}{% endif %}{% if post.date-end %}
-DTEND;TZID=America/New_York:{{ post.date-end | date: "%Y%m%dT%H%M00" }}{% else %}
+DTEND;TZID=America/New_York:{{ post.date-end | date: "%Y%m%dT%H%M%S" }}{% else %}
 DTEND:{{ post.date | date: "%Y%m%d" }}{% endif %}{% if post.rrule %}
-RRULE:{{post.rrule}}{% endif %}
-ORGANIZER;CN="{{site.title}}":MAILTO:{{site.email}}
-SUMMARY:{{post.title}}
-DESCRIPTION:{{post.excerpt | strip_html | newline_to_br | replace: "<br />", " " | strip_newlines | strip}}
+RRULE:{{ post.rrule | strip_newlines }}{% endif %}
+ORGANIZER;CN="{{ site.title }}":MAILTO:{{ site.email }}
+SUMMARY:{{ post.title | escape }}
+DESCRIPTION:{{ post.excerpt | strip_html | newline_to_br | replace: "<br />", " " | strip_newlines | escape }}
 CLASS:PUBLIC{% if post.location %}
-LOCATION:{{post.location}}{% endif %}{% if post.redirect %}
-URL:{{post.redirect}}{% elsif post.categories.last != "meetings-meetups" %}
-URL:{{site.url}}{{site.baseurl}}{{post.url}}{% else %}
-URL:{{site.url}}{{site.baseurl}}/get-involved{% endif %}{% if post.categories.last == "meetings-meetups" %}
```

Comment:
Is there a reason why these variations on the URL were removed? The warnings from the RFC checker were mostly timezone and line length related, so I'm not seeing a reason to change this